contracts/Dropper/DropperFacet.sol

```diff
@@ -111,6 +111,11 @@ contract DropperFacet is
         ds.TerminusAdminPoolID = terminusAdminPoolID;
     }
 
+    function getTerminusAdminInfo() external view returns (address, uint) {
```

Comment:
@ShivaLaguna : Can you rename this function to be consistent with our other contracts? See for example Garden of Forking Paths: https://github.com/bugout-dev/engine/blob/main/contracts/mechanics/garden-of-forking-paths/GardenOfForkingPaths.sol#L204

Reply:
Name should be `adminTerminusInfo`.